docs/source/pythonapi/examples/post-processing.rst

```diff
@@ -0,0 +1,13 @@
+.. _notebook_post_processing:
+
+===============
+Post Processing
+===============
+
+.. only:: html
+
+    .. notebook:: post-processing.ipynb
```

Comment:
This is really just the name of the notebook (if you have a better non-generic name, feel free to suggest). In the index page for the Python API, you'll see it's listed alongside the pandas dataframes and tally arithmetic notebooks.